packages/datagrid/src/containers/datagrid.connect.js

```diff
@@ -0,0 +1,26 @@
+import { cmfConnect } from '@talend/react-cmf';
+
+import Container from './datagrid.container';
+
+export function mapStateToProps(state, ownProps) {
+	const props = {};
+	if (ownProps.source) {
+		props.data = state.cmf.collections.getIn(ownProps.source.split('.'));
```

Comment:
please remove this it can be written the following way now:
```javascript
<DataGrid dataExpression={{id: 'cmf.collections.get', args:['foo.bar.data', []]} />
```